lib/src/record_replay/manifest.dart

```diff
@@ -47,25 +45,28 @@ class Manifest {
   /// [toJson]), a [FormatException] will be thrown.
   factory Manifest.fromJson(String json) {
     List<Map<String, dynamic>> decoded =
-        new JsonDecoder().convert(json).cast<Map<String, dynamic>>();
-    Manifest manifest = new Manifest();
+        (JsonDecoder().convert(json) as List<dynamic>)
```

Comment:
It's not typical to construct a `JsonDecoder` manually.

```suggestion
        (jsonDecode(json) as List<dynamic>)
```